schemas/schema_helpers.go

```diff
@@ -178,6 +178,33 @@ func diffSuppressInterfaceListOrder(mapKey string) schema.SchemaDiffSuppressFunc
 	}
 }
 
+// diffSuppressAppInterfaceListOrder checks whether we have the newly created AppInterface
+// or not. It suppress any differences after an AppInstance is created.
+// AppInterfaces are never updated after they are created
+func diffSuppressAppInterfaceListOrder(mapKey string) schema.SchemaDiffSuppressFunc {
```

Comment:
Maybe call this `supressAppInterfaceAfterCreation` or similar?

Reply:
ok, sounds good